### DISCUSSION TITLE
3D Slicer Auto Segmentation greyed out

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1233] [Q&A]
Hi in 3D Slicer in none of the models the Auto Segmentation pull down remains blank. **I am trying to use the Use Case 2: Bundle with Customized Scripts for Renal Substructure Segmentation** in the QUICKSTART menu to segment the kidney.
The script to start server seems to run fine. The issue I'm having seems slightly different to prior issues on this site.
The photo is from radiology app but the other apps do exactly the same thing.
The server loads, the model seems to load in the Active Learning section but nothing in Auto Segmentation

![slicer screen](https://user-images.githubusercontent.com/65087913/209869759-3ce290f0-68ed-41c5-b68a-d78092cec03c.jpg)

Any help most appreciated.

**Comment by amitjc** [comment]
The Auto Segmentation submodule of Monailabel is operational in version 5.2.0 on Windows.

**Reply by borantolic** [reply]
I also have the Auto Segmentation disabled. I can train on spleen images, but I cannot seem to get the system to auto-segment the CT. I use Slicer 5.2.1 for Mac.
How do I get the Autosegmentation to work?

**Reply by diazandr3s** [reply]
Hi @amitjc,

This is a bit strange. You should be able to run Auto Segmentation.
Just to rule out other issues, could you please try the whole-body CT segmentation model and let us know?

Here you can find the radiology folder (https://drive.google.com/drive/folders/17eJan-8_oNCnZyJk8B9zLQpwaOjtDuKi?usp=share_link) and the command to run it is: 

**_monailabel start_server --app radiology_full_ct-upgraded-HYBRID/ --studies PATH_TO_IMAGES/ --conf models segmentation_full_ct_**

It also includes liver segmentation. Please let us know how that goes

BTW, this **_Wednesday, the 25th of Jan 2023_** we're running a workshop for MONAI Label. It is free to participate and you can try different pretrained models on the AWS platform: https://projectweek.na-mic.org/PW38_2023_GranCanaria/MONAILabel_Workshop.html

**Reply by abhishekmarkad** [reply]
> I also have the Auto Segmentation disabled. I can train on spleen images, but I cannot seem to get the system to auto-segment the CT. I use Slicer 5.2.1 for Mac. How do I get the Autosegmentation to work?

Facing same issue for Radiology app , in auto segmentation no model present , any help .
Thank you

**Reply by tangy5** [reply]
Can you try re-install the plugin?  Or, it would be safe to make a clean install with both 3D Slicer and MONAI label plugin, since previously we see some users are facing this issue, and it finally turns out that the plugin files are somehow corrupted. 

Another direction to debug, and double check is the inferer, in the models' inferer in not correctly set, the auto segmentation button will gery out since there are no models/inferer available. 

Thanks

**Reply by diazandr3s** [reply]
> > I also have the Auto Segmentation disabled. I can train on spleen images, but I cannot seem to get the system to auto-segment the CT. I use Slicer 5.2.1 for Mac. How do I get the Autosegmentation to work?
> 
> Facing same issue for Radiology app , in auto segmentation no model present , any help . Thank you

Hi @abhishekmarkad,

Thanks for reporting this.
Can you please provide more information so we can further investigate the reason for this? 
Did this happen after fetching the Next Sample from Slicer?
